repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/pipeline/Launcher.java

```diff
@@ -243,6 +251,20 @@ private void initNotifiers() {
         this.notifiers.add(new FixerBuildNotifier(notifierEngines));
     }
 
+    private List<String> getListOfProjectsToIgnore() {
+        List<String> result = new ArrayList<>();
+        if (this.config.getProjectsToIgnoreFilePath() != null) {
+            try {
+                for (String line : Files.readAllLines(new File(this.config.getProjectsToIgnoreFilePath()).toPath())) {
+                    result.add(line.trim().toLowerCase());
+                }
+            } catch (IOException e) {
+                throw new RuntimeException("Error while reading projects to be ignored from file: " + this.config.getProjectsToIgnoreFilePath(), e);
```

Comment:
I don't agree: in this case we should just log an error and use an empty list: it's overkill to throw a runtime exception for an optional feature like this one.

Reply:
Done.